apis/v1alpha1/generator.yaml

```diff
@@ -21,4 +21,50 @@ resources:
         code: customPreCompare(delta, a, b)
       sdk_create_post_set_output:
         template_path: hooks/workspace/sdk_create_post_set_output.go.tpl
+  RuleGroupsNamespace:
+    shortNames:
+      - rulegroup
+      - rulegroups
```

Comment:
nit: I'm not really sure about using those short names, maybe go for `rgn` instead?